### DISCUSSION TITLE
Renaming the Doks folder

### DISCUSSION THREAD

**Discussion by rajtilakjee** [thuliteio/doks#1011] [Q&A] [answered]
I could change the menu items, but if you notice the folder is still named the same and it shows in the breadcrumbs. I tried changing that but I ended up messing everything locally. Could you please help me with that @h-enk? Here's the link to my site:

https://devmaps.netlify.app/

**Comment by rajtilakjee** [comment]
@h-enk first of all, apologies for bothering you over and over again. However, let me assure you that I am trying to solve the problems as much as possible from my end. So, since the above post, I followed the following discussions:

https://github.com/h-enk/doks/discussions/977
https://github.com/h-enk/doks/discussions/1004

I was able to solve a part of my problem. As you would see, I was able to add a new folder instead of Docs at the top, and add content to it. It worked. But right now I don't have a sidebar on the content level. I tried cleaning the cache using `npm run clean` but that didn't work in my case. I have been bothering members of the Hugo Discord server and was able to come this far using their help. But I am really stuck here and would need your divine intervention.

Thanks in advance.

**Reply by h-enk** [reply]
No worries. 

Did you copy and paste the templates in `./layouts/docs/` to `./layouts/roadmaps/` — see also https://github.com/h-enk/doks/discussions/977?

**Reply by rajtilakjee** [reply]
Yes, now I did. And it worked. Thank you so very much!

**Reply by rajtilakjee** [reply]
Can you write this as a separate comment so that I can mark it as an answer.

**Comment by h-enk** [comment, marked as answer]
Did you copy and paste the templates in ./layouts/docs/ to ./layouts/roadmaps/ — see also https://github.com/h-enk/doks/discussions/977?
